com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -444,6 +444,7 @@ internal void InitializeVariables()
                     }
 
                     instance.SetNetworkBehaviour(this);
+                    instance.Name = sortedFields[i].Name;
```

Comment:
oof, we already do (one-off) runtime-reflection for NetVars. I didn't know that...